powersimdata/input/abstract_grid.py

```diff
@@ -31,6 +31,14 @@ def __init__(self):
         self.branch = pd.DataFrame()
         self.storage = storage_template()
 
+
+class AbstractGridCSV(AbstractGrid):
+    """Grid Builder."""
+
+    def __init__(self):
+        """Constructor"""
+        super().__init__()
```

Comment:
Do we need this if it only calls `super`?

Reply:
No we don't

Reply:
Indeed. If `__init__` behavior are the same with parent, it can be inherited without explicit calls: https://stackoverflow.com/questions/54911999/does-a-subclass-need-to-initialize-an-empty-super-class